Vezba/Program.cs

```diff
@@ -170,5 +204,124 @@ public static void Lists()
+        {
+            Console.WriteLine(a * b);
+        }
+
+        //5. __ne vraca nista i ima pet ulaznih parametara, prva dva tipa int, ostali tipa string.
+        //Funkcija ne radi nista.
+        public static void FuncFiveDoNothng(int a, int b, string s1, string s2, string s3) { }
+
+        //6. __ne vraca nista i ima 6 ulaznih parametara,  prvi tipa int, drugi string, treci char,
+        //cetvrti string, peti int, sesti bool. Funkcija ne radi nista.
+        public static void FuncSixDoNothing(int a, string s1, char c1, string s2, int b, bool bolean) { }
+
+        //7. __ne vraca nista i ima tri ulazna parametara, sva tri tipa string. 
+        //Funkcija ispisuje u konzoli sva tri prosledjena parametra.
+        public static void FunThreeConsol(string s1, string s2, string s3)
+        {
+            Console.WriteLine(s1, s2, s3);
+        }
+
+        //8. __koja vraca zbir tri ulazna parametra koja su tipa int.
+        public static int FunInteger3(int a, int b, int c)
+        {
+            int res = a + b + c;
+            return res;
+        }
+
+        //9. __koja vraca zbir tri ulazna parametra koja su tipa long.
+        public static long FunLong(long l1, long l2, long l3)
+        {
+            return l1 + l2 + l3;
+        }
+
+        //10. __koja vraca da li je prvi ulazni parametar veci od drugog.
+        public static bool FirstGreterThenSecond(int num1, int num2)
+        {
+            return num1 > num2;
+        }
+
+        //11. __koja vraca da li je prvi ulazni parametar manji od drugog.
+        //Prvi parametar je tipa int, drugi isto.
+        public static bool FirstLessThenSecond(int par1, int par2)
+        {
+            return par1 < par2;
+        }
+
+        //12. __koja vraca da li je prvi ulazni parametar veci od treceg.
+        //Prvi parametar je tipa int, drugi tipa string, treci tipa int.
+        public static bool FirstGreterThenThrd(int par1, string s2, int par3)
+        {
+            return par1 > par3;
+        }
+
+        //13. __koja vraca vrednost ulaznog parametra uvecanu za jedan.
+        //Ulazni parametar je tipa int.
+        public static int FunIncrease(int increase)
+        {
+            return increase++;
+        }
+
+        //14. __koja vraca niz stringova. Ulazni parametri su dva stringa.
+        //Izlazni niz treba da sadrzi oba stringa.
+        public static string[] FunStringArray(string s1, string s2)
+        {
+            string[] twoStr = new string[2];
+            twoStr[0] = s1;
+            twoStr[1] = s2;
+            return twoStr;
+        }
+
+        //15. __koja vraca listu stringova. Ulazni parametri su 10 stringova.
+        //Izlazna lista treba da sadrzi svih deset stringova.
+        public static List<string> FunListOfString(string v1, string v2, string v3, string v4,
+        string v5, string v6, string v7, string v8, string v9, string v10)
+        {
+            List<string> Student = new List<string>(10);
```

Comment:
There is no need to specify capacity. 
Just new List<string>() is enough.
It will initialize list to default capacity and will extend it if needed.